modules/cluster/variables.tf

```diff
@@ -21,6 +21,12 @@ variable "enable_logs_storage" {
   default = true
 }
 
+variable "expire_logs_after_day" {
+  description = "Number of days objects in the logs bucket are stored"
+  type = number
+  default = 90
```

Comment:
No need to default this variable as the top level variable will map to this and pass through the number value.